nixos/modules/services/networking/ddclient.nix

```diff
@@ -102,6 +102,22 @@ in
           Method to determine the IP address to send to the dynamic DNS provider.
         '';
       };
+
+      verbose = mkOption {
+        default = true;
```

Comment:
Might be that it also depends on which dynamic DNS provider you use. It might log more for others…